trino/client.py

```diff
@@ -281,6 +287,13 @@ def http_headers(self) -> Dict[str, str]:
         transaction_id = self._client_session.transaction_id
         headers[constants.HEADER_TRANSACTION] = transaction_id
 
+        if self._client_session.extra_credential is not None and \
+                len(self._client_session.extra_credential) > 0:
+            # HTTP 1.1 section 4.2 combine multiple extra credentials into a
+            # comma-separated value
+            headers[constants.HEADER_EXTRA_CREDENTIAL] = \
+                ", ".join(["{}={}".format(*tup) for tup in self._client_session.extra_credential])
```

Comment:
Please encode the value. Otherwise, the requests will fail when it needs encoding (e.g. `,`). Also, please add the relevant test if possible.

Reply:
> Please encode the value.

This comment isn't yet addressed.